src/components/SecurityStep.jsx

```diff
@@ -77,4 +80,82 @@ const SecurityStep = ({ onSkip }) => {
   )
 }
 
+const OIDCSecurityStep = ({ onNext }) => {
+  const { t } = useI18n()
+
+  const { isVaultConfigured } = useStepsContext()
+
+  return (
+    <VerticallyCentered>
+      <Wrapper>
+        <NarrowContent>
+          <Stack>
+            <img src={strongPasswordIcon} alt="" width={204} height={137} />
+            <MainTitle>
+              {isVaultConfigured
+                ? t('SecurityStepOIDC.title-configured')
+                : t('SecurityStepOIDC.title')}
+            </MainTitle>
+            <Stack spacing="xl">
+              <Text>
+                {isVaultConfigured
+                  ? t('SecurityStepOIDC.description-configured')
+                  : t('SecurityStepOIDC.description')}
+              </Text>
+              {isVaultConfigured ? (
+                <div>
+                  <ChangePasswordLink
+                    extension="full"
+                    theme="secondary"
+                    label={t('UpdateCozyPassPassword')}
+                    successRoute="#/installation/configureExtension"
+                    cancelRoute="#/installation/configureExtension"
+                  />
+                </div>
+              ) : (
+                <>
+                  <SetVaultPasswordForm onSuccess={onNext} />
+                  <Card>
+                    <UnorderedList className="u-ta-left u-mv-0">
+                      <ListItem
+                        dangerouslySetInnerHTML={{
+                          __html: snarkdown(
+                            t('SecurityStep.advices.strong_passphrase')
+                          )
+                        }}
+                      />
+                      <ListItem
+                        dangerouslySetInnerHTML={{
+                          __html: snarkdown(t('SecurityStep.advices.memorize'))
+                        }}
+                      />
+                      <ListItem>
+                        <span
+                          dangerouslySetInnerHTML={{
+                            __html: snarkdown(t('SecurityStep.advices.our_tip'))
+                          }}
+                        />
+                        <PasswordExample password="Cl4ude€st1Nu@ge" />
```

Comment:
Didn't we decide something about this?

Reply:
I haven't had a clear go on this I think, @poupotte ?

Reply:
After discussion with Celine, we suggest : "Claude est le meilleur nuage français !"  / "Claude is the best french cloud !", if it's ok for you ? 
I'll create a card for Settings.

Reply:
<img width="474" alt="image" src="https://user-images.githubusercontent.com/465582/112173704-1aa26c80-8bf6-11eb-8e56-0633c979211f.png">

La coloration ne sert plus à grand chose maintenant. Je le garde en local pour le moment.

Reply:
Oui, effectivement, on peut la retirer, ça n'a plus de sens avec ce nouvel exemple

Reply:
OK, j'enleverais la coloration quand https://github.com/cozy/cozy-ui/pull/1777 sera mergée (capacité à enlever la coloration sur l'exemple de mot de passe).